src/imitation/algorithms/adversarial.py

```diff
@@ -281,8 +284,13 @@ def _build_disc_feed_dict(self, *,
     assert n_gen == len(gen_acts)
     assert n_gen == len(gen_next_obs)
 
+    # Normalize expert observations to match generator observations.
+    assert isinstance(self.venv_train_norm, VecNormalize)
+    expert_obs_norm = self.venv_train_norm._normalize_observation(
```

Comment:
`_normalize_observation` as a side-effect updates `self.obs_rms`.

Maybe we should change Stable Baselines to (a) make `_normalize_observation` part of public API (document, rename to remove trailing `_`) and (b) take a boolean flag for whether `obs_rms` should be updated?

Alternative is to write our own helper method that does this, or toggle `self.venv_train_norm.training` (but that's ugly!)

Reply:
Wrote a helper method for now -- I'll make an issue in SB to make normalize_observation public.

Reply:
https://github.com/hill-a/stable-baselines/issues/602